### PR TITLE
Reorder the sequence of includes

### PR DESCRIPTION
**Pull request by sung-su**

### CODE REVIEW

**Comment by WonyoungChoi** [comment]
LGTM

**Comment by myroot** [comment]
LGTM

File: runtime/browser/web_application.cc [hunk 1]
```diff
@@ -16,31 +16,31 @@
 
 #include "runtime/browser/web_application.h"
 
+#include <algorithm>
```
Comment (WonyoungChoi) [inline runtime/browser/web_application.cc]: algorithm is c++ header, move to after c headers.

File: extensions/renderer/xwalk_extension_client.cc [hunk 2]
```diff
@@ -5,14 +5,13 @@
 
 #include "extensions/renderer/xwalk_extension_client.h"
 
-#include <unistd.h>
-#include <glib.h>
 #include <gio/gio.h>
+#include <glib.h>
 #include <string>
```
Comment (WonyoungChoi) [inline extensions/renderer/xwalk_extension_client.cc]: string is c++ header, move to after c headers.

**Comment by WonyoungChoi** [comment]
sorry, I canceled my LGTM because I found some mistakes.
Run ./tools/dolint.sh before PR. It is helpful for finding these mistakes.

**Pull request merged**